stylesheets/abstracts/_variables.scss

```diff
@@ -77,14 +70,15 @@ $white: hsl(0, 0%, 100%) !default;
 // Rose Violet
 $rose-violet: hsl(0, 100%, 95%) !default;
 
-// Typography
+// Fonts
 $sans-serif-stack: 'Lato', sans-serif !default;
 $serif-stack: 'Manuale', serif !default;
-$font-weight-light: 300 !default;
+
+// Font weight
 $font-weight-normal: 400 !default;
 $font-weight-bold: 700 !default;
 $font-weight-extra-bold: 800 !default;
-$font-weight-heavy: 900 !default;
+$font-weight-black: 900 !default;
```

Comment:
wondering if `black` is misleading for developers because it implies color?

Reply:
Yeah, perhaps we shouldn't change that. I saw that the storybook documentation that was already in place used "black" and I referenced [this weight name mapping](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping), which indicates to me we could go either way so maybe don't fix what isn't broken and keep heavy.